### ISSUE TITLE
Updating reference once a preprint is published?

### ISSUE THREAD

**Issue by sje30** [codecheckers/discussion#7]
The covid-uk study (2010-008) has now been published; how do we cite to the paper, as a follow up?

https://doi-org.ezp.lib.cam.ac.uk/10.1016/S2468-2667(20)30133-X

**Comment by nuest** [comment]
Good point. Do we need separate fields for preprint DOI (which was checked) and the published paper? Not in this case, but differences might occur, tough that could be the easier case: if there are differences, then we don't update anything.

If there are no differences, as is the case with `2010-008`. The [`codecheck.yml`](https://github.com/codecheckers/covid-uk/blob/master/codecheck.yml#L14) currently only has

```yml
  reference: <https://cmmid.github.io/topics/covid19/control-measures/report/uk_scenario_modelling_preprint_2020_04_01.pdf>
```

- Why not update the reference field to use the DOI?
- Can the `source` field be used to describe the history?
- What about updating the Zenodo Metadata to link to the published paper?

OTOH, maybe we deliberately don't reference the paper at all, because the CODECHECK was only for the preprint version of code and data, and we should only have a reference to the article if the CODECHECK was a "journal check" (not a "community codecheck") ?

More thoughts than answers at this point.
